glusterd2/transaction/context.go

```diff
@@ -119,7 +113,7 @@ func (c *Tctx) Get(key string, value interface{}) error {
 	storeKey := c.prefix + "/" + key
 	r, err := store.Store.Get(context.TODO(), storeKey)
 	if err != nil {
-		c.log.WithError(err).WithField("key", storeKey).Error("failed to get value from store")
+		c.log.WithError(err).WithField("key", storeKey).Error("failed to get value from transaction context")
```

Comment:
I think here the key is being fetched from the store and not in transaction context